src/components/GradebookFilters/AssignmentGradeFilter/index.jsx

```diff
@@ -40,22 +42,28 @@ export class AssignmentGradeFilter extends React.Component {
       localAssignmentLimits: { assignmentGradeMax, assignmentGradeMin },
     } = this.props;
     return (
-      <div className="grade-filter-inputs">
-        <PercentGroup
-          id="assignmentGradeMin"
-          label={<FormattedMessage {...messages.minGrade} />}
-          value={assignmentGradeMin}
-          disabled={!this.props.selectedAssignment}
-          onChange={this.handleSetMin}
-        />
-        <PercentGroup
-          id="assignmentGradeMax"
-          label={<FormattedMessage {...messages.maxGrade} />}
-          value={assignmentGradeMax}
-          disabled={!this.props.selectedAssignment}
-          onChange={this.handleSetMax}
-        />
-        <div className="grade-filter-action">
+      <>
+        <Row>
+          <Col xs={5} className="pr-0">
```

Comment:
[curious] Is there another implementation approach here besides using `Row` and `Col`? It's generally a code smell to need to mess with the horizontal padding on columns in the grid system.

Would treating a container `div` as `d-flex` be another reasonable solution here?